static/sass/_pattern_careers-modal.scss

```diff
@@ -23,9 +23,15 @@
       border-bottom: 0;
       display: block;
       margin-bottom: $spv--medium;
+      padding-top: 0;
+
+      &::after {
+        height: 0;
+      }
```

Comment:
Isn't this going to remove lines for all modals? I can't see the application modals, but I suppose those still follow the old styling.

Reply:
It will but this is the trouble with not using Vanilla or more importantly a central design system.

Reply:
Opened Github issue as suggested by Lyubo https://github.com/canonical/vanilla-framework/issues/4883